evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -186,13 +186,14 @@ def _create_naive_pipelines(self, use_features=False):
         estimators = self._naive_estimators()
         pipelines = [
             make_pipeline(
-                self.X,
-                self.y,
-                estimator,
-                self.problem_type,
+                X=self.X,
+                y=self.y,
+                estimator=estimator,
+                problem_type=self.problem_type,
                 sampler_name=self.sampler_name,
                 extra_components=feature_selector,
                 extra_components_position="after_preprocessing",
+                parameters=self._pipeline_params,
```

Comment:
Was this a previous bug?

Reply:
@freddyaboulton  the pipeline parameters should have been passed in through [line 201](https://github.com/alteryx/evalml/pull/3129/files#diff-3968539fa92e26ee6c6a8ae303303ee0aed6cae4a3afa3d1007d26b4b7b0c09fR201) where it's added in `_transform_parameters()`. 

@chukarsten did you notice that the pipeline parameters were not passing in?